cmd/alluxio/main.go

```diff
@@ -17,12 +17,15 @@ package main
 
 import (
 	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/arl/statsviz"
```

Comment:
How about using "net/http/pprof" ? Kubernetes is also using this.

Reply:
Thanks very much for your good advice. I will use pprof to implement it.